__tests__/app.test.js

```diff
@@ -181,4 +181,27 @@ describe('GET /api/articles/:article_id (comment count)', () => {
             });
         });
     });
+});
+
+describe('GET /api/articles', () => {
+    test('status: 200, respond with array of article objects with correct properties', () => {       
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then( ({body}) => {
+            expect(body.articles).toEqual(
+                expect.arrayContaining([
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        topic: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(Number),
+                    }),
+                 ])
+            )
+        })
+    })
 });
```

Comment:
We want the data to have a default sort order as per the ticket so you will need a test for this